CHANGELOG.rst

```diff
@@ -1,9 +1,22 @@
 0.10
 ----
 
-- Use YYMMDD tag in nightly build
+- Change the updated new_worker_spec API for upstream (#128) `Matthew Rocklin`_
+- Update TOTAL_MEMORY to match new distributed MEMORY_LIMIT (#131) `Peter Andreas Entschev`_
+- Bum Dask requirement to 2.4 (#133) `Matthew Rocklin`_
+- Use YYMMDD tag in nightly build (#134) `Markku Luukkainen`_
+- Automatically determine CPU affinity (#138) `Peter Andreas Entschev`_
+- Fix full memory use check testcase (#139) `Sangeeth Keeriyadath`_
+- Use pynvml to get memory info without creating CUDA context (#140) `Peter Andreas Entschev`_
+- Pass missing local_directory to Nanny from dask-cuda-worker (#141) `Peter Andreas Entschev`_
+- New worker_spec function for worker recipes (#147) `Peter Andreas Entschev`_
+- Add new Scheduler class supporting environment variables (#149) `Peter Andreas Entschev`_
+- Support for TCP over UCX (#152) `Peter Andreas Entschev`_
 
+.. _`Matthew Rocklin`: https://github.com/mrocklin
+.. _`Peter Andreas Entschev`: https://github.com/pentschev
 .. _`Markku Luukkainen`: https://github.com/mluukkainen
+.. _`Sangeeth Keeriyadath`: https://github.com/ksangeek
```

Comment:
We don't need to repeat names for every section.  Typically I just include the links at the bottom.  See https://raw.githubusercontent.com/dask/distributed/master/docs/source/changelog.rst for an example

Reply:
So, for example, you don't need to add your name or my name here because they are already defined below.

Although what you have here is also ok, if you prefer it

Reply:
Makes sense, let me change that.